src/www/ui/api/documentation/openapi.yaml

```
@@ -3125,6 +3168,41 @@ components:
           type: string
           example:
             "Main group"
+    GetBulkHistory:
+      type: object
+      properties:
+        bulkId:
+          type: integer
+          description: The bulk id
+          example: 1
+        clearingEventId:
+          type: integer
+          description: The job id associated with the bulk
```

No need to be in detail :-)
```suggestion
          description: The event id associated with the bulk
```